src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+@import url(./vars.css);
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0;
+  padding-inline-start: 0;
+}
+
+.nav_item {
+  display: flex;
+  list-style-type: none;
+  margin-right: 20px;
+  text-transform: uppercase;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
```suggestion
```
![image](https://user-images.githubusercontent.com/99331793/201341084-9933ea96-dfae-4c97-a3f6-e1af9bc7b797.png)

make it work without it